README.rst

```diff
@@ -37,9 +37,13 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy or on making code contributions, please see the ``neuroimaging`` mailing list:
 
-    http://projects.scipy.org/mailman/listinfo/nipy-devel
+    https://mail.python.org/mailman/listinfo/neuroimaging
+
+Github issues may also be used to report bugs, or to see a list of known bugs and planned changes:
```

Comment:
Sorry, grammar tweak: No comma, since the last part wouldn't be a full sentence on its own, and the list is now two items long.